### DISCUSSION TITLE
10 x 10 stash setting

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#289] [Q&A] [answered]
Hi. after today's update, 10 x 10 stash doesn't work.(single play)
I just know edit Storage.js
this.StashY = me.gametype === 0 ? 4 : 8;
this.Stash = new Container("Stash", 6, this.StashY, 7);
to 10 x 10
where should I edit more?

**Comment by isid0re** [comment, marked as answer]
Update to the latest push and your customization should be fine

**Reply by mirakhany** [reply]
Thank you for quick update!
No problem with 10 x 10!
